web/src/actions/index.js

```diff
@@ -122,6 +122,9 @@ export function submitSentences(language, sentences, source) {
     dispatch(submitSentencesSuccess(results.sentences.slice(0)));
     const errorsWithSentenceInfo = results.errors.filter((error) => error.sentence);
     dispatch(submitSentencesFailureSingle(errorsWithSentenceInfo));
+    if( !state.app.languages.includes( language ) ) {
+      dispatch(addLanguage( language ));
+    }
```

Comment:
Could you remove the spaces here? For this code base here, we use the following convention:

* Space after `if`, no space inside if -> `if (!state.app.languages.includes(language))`
* No spaces in parameter lists -> `addLanguage(language)`

Of course that's completely preference based :)

Reply:
Of course, PR coming right away